src/circuit.rs

```diff
@@ -72,7 +680,7 @@ impl VerifyingKey {
     /// Builds the verifying key.
     pub fn build() -> Self {
         let params = halo2::poly::commitment::Params::new(K);
-        let circuit = Circuit {}; // TODO
+        let circuit: Circuit = Default::default(); // TODO
```

Comment:
Isn't this correct? Why is it TODO? (Similarly below.)

Reply:
I added that `TODO` when I wrote the circuit stubs, because an empty struct was obviously not correct, and I expected there might be some strucural values that needed preparing. But if `Circuit` implements `Default` then this is definitely correct.